### PR TITLE
Update ediscovery-download-documents-from-review-set.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated limits to the document 

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11175

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ediscovery-download-documents-from-review-set.md [hunk 1]
```diff
@@ -27,4 +27,6 @@ Download offers a simple way to download content from a review set in native for
 
 To download content from a review set in an eDiscovery (Premium) case, start by selecting the files you want to download then select **Action items** > **Download**.
 
+Total file size or maximum number of documents downloaded from a review set is 3 MB or 50 documents respectively.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-download-documents-from-review-set.md]:
```suggestion
The total file size or the maximum number of documents downloaded from a review set is 3 MB or 50 documents respectively. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder.

**Comment by yogkumgit** [comment]
Closing as per the comment in #11175

**Pull request closed without merging**